docs/Getting Started/Installation/index.md

```diff
@@ -189,3 +189,21 @@ from within the JavaEE directory and a JAR should be generated in the build/libs
 Glassfish 5.0.0 only works on JDK 8 and lower.
 !!!
 !@
+
+@![javascript]
+The SDL JavaScript Suite comes with prebuilt bundles of the library. There is a vanilla JavaScript distribution of the library as well as one for Node.js. They are located in the lib/js/dist and lib/node/dist directories respectively. To use one of these bundles, include one of the following somewhere in your project:
+
+        const SDL = require('lib/js/SDL.min.js');
```

Comment:
For the JS dist, it will likely be imported in the browser and so `require` wouldn't work, as that's a Node specific import syntax. Recommend clarifying that or changing the line to show how to do it in the browser: `<script src='lib/js/SDL.min.js'></script>`